client/my-sites/plans/jetpack-plans/product-lightbox/included-product-list.tsx

```diff
@@ -0,0 +1,59 @@
+import { TranslateResult, useTranslate } from 'i18n-calypso';
+import getProductIcon from '../product-store/utils/get-product-icon';
+import slugToSelectorProduct from '../slug-to-selector-product';
+
+type IncludedProductsProps = {
+	products: ReadonlyArray< string >;
+	descriptionMap: Record< string, TranslateResult >;
+};
+
+type IncludedProductListItemProps = {
+	productSlug: string;
+	descriptionMap: Record< string, TranslateResult >;
+};
+
+const IncludedProductListItem: React.FC< IncludedProductListItemProps > = ( {
+	productSlug,
+	descriptionMap,
+} ) => {
+	const product = slugToSelectorProduct( productSlug );
+	return (
+		<div className="product-lightbox__included-product-list-item">
+			<div className="product-lightbox__included-product-list-item-icon">
+				<img alt="" src={ getProductIcon( { productSlug, light: true } ) } />
+			</div>
+
+			<div className="product-lightbox__included-product-list-item-content">
+				<h1 className="product-lightbox__included-product-list-item-title">
+					{ product?.shortName }
+				</h1>
+
+				<p className="product-lightbox__included-product-list-item-description">
+					{ descriptionMap[ productSlug ] }
+				</p>
+			</div>
+		</div>
+	);
+};
+
+const IncludedProductList: React.FC< IncludedProductsProps > = ( { products, descriptionMap } ) => {
+	const translate = useTranslate();
+
+	return (
+		<div className="product-lightbox__included-product-list">
+			<h1 className="product-lightbox__included-product-list-label">
```

Comment:
Since the modal heading is a `h2`, this should be a `h3`.

Reply:
good catch @monsieur-z 👍